tools/gradle/plugin/src/functionalTest/kotlin/com/zeroc/ice/SliceToolsJavaPluginFunctionalTest.kt

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) ZeroC, Inc.
+
+package com.zeroc.ice
```

Comment:
Should we use a subpackage name like `com.zeroc.ice.slice` or `com.zeroc.ice.slice.tools`?

Putting these classes directly com.zeroc.ice looks odd to me.

Reply:
We can but it would be part of the plugin ID, that users have to use to apply the plugin.

Reply:
Good them, we just need to settle on what names to use.